storage/sql/coresql/tree_storage.go

```diff
@@ -86,18 +86,13 @@ func (t *treeTX) getSubtrees(treeRevision int64, nodeIDs []storage.NodeID) ([]*s
 		return nil, nil
 	}
 
-	stmt, args, err := t.ts.wrap.GetSubtreeStmt(t.tx, t.treeID, treeRevision, nodeIDs)
-	if err != nil {
-		return nil, err
-	}
-	defer stmt.Close()
-
-	rows, err := stmt.Query(args...)
+	stmt, rows, err := t.ts.wrap.GetSubtrees(t.tx, t.treeID, treeRevision, nodeIDs)
```

Comment:
An option here might be to pass in a `func(...)` here which gets called once for each row, then you could `Close()` the stmt and rows inside the call itself.